### PR TITLE
Ft/docker desktop

### PR DESCRIPTION
**Pull request by max8github**
## Purpose

Adds instructions for Docker Desktop, improves Kubernetes deployment instructions overall, adds Kubernetes namespace, removes unused content, see issue #15.

## References
Feature request #15, #5 (left over `project/build.properties`)

References #15

## Changes

* Corrected `.gitignore`
* Removed unused class `ClusterWatcher`
* Removed sbt related files (this is a maven project)
* Added Kubernetes namespace `appka-1` (handy)
* Opened `8080` port in Kubernetes service for browser 'hello world' displaying
* Edited docs accordingly (`README`)
* Added test script accordingly
* Added documentation for test scripts
* Fixed Travis file accordingly

## Background Context

Docker desktop is a popular Kubernetes cluster implementation handy to use on a Mac desktop.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,10 @@
...
+target
+
+# IDE #
+######################
+nbproject
+.idea
+nbactions.xml
+*.iml
+private
+classes
```
Comment (raboof) [inline .gitignore]: (some of those you could add to your global gitignore so you don't need them in each project, https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore) - but they're :+1: here as well

File: README.md [hunk 2]
```diff
@@ -12,16 +12,54 @@ coordination between nodes is necessary, this is where the
 become interesting and it is worth consider making the nodes form an Akka
 Cluster.
 
+## Kubernetes Instructions
+    
+### Docker Desktop for Kubernetes
+If you use Kubernetes on Docker Desktop, after turning it on, you should first issue:
```
Comment (raboof) [inline README.md]: Perhaps link to https://www.docker.com/products/docker-desktop and clarify that this only applies to Mac and Windows users?

File: README.md [hunk 3]
```diff
@@ -12,16 +12,54 @@ coordination between nodes is necessary, this is where the
...
+
+    cd akka-sample-cluster-kubernetes-java/kubernetes
+    
+    export KUBECONFIG=~/.kube/config
+    kubectl config set-context docker-desktop
+    
+A script does all this is `scripts/test_docker_desktop.sh`. To run it, do:
+
+    cd akka-sample-cluster-kubernetes-java
+    chmod +x scripts/test*
```
Comment (raboof) [inline README.md]: I think you can commit the +x bit so a chmod shouldn't be needed here, right?

File: .gitignore (hunk 1 above)
Comment (max8github) [reply .gitignore]: yes, unsure why i did this, actually, i have the same set up with a global gitignore: leaving `target` only.

**Pull request merged**